### ISSUE TITLE
Disable view

### ISSUE THREAD

**Issue by adrielcafe** [jaredrummler/Material-Spinner#10]
Hi!
How to disable MaterialSpinner for prevent that user change the selected value?
I tried:
```xml
<com.jaredrummler.materialspinner.MaterialSpinner
                ...
                android:enabled="false"
                android:clickable="false"
                android:focusable="false"
                android:focusableInTouchMode="false"/>
```
But PopupWindow continues to show.

**Comment by jaredrummler** [comment]
This is fixed in 1.0.7

**Comment by adrielcafe** [comment]
Thanks man!
